src/views/AddLiquidity/index.tsx

```diff
@@ -84,6 +85,12 @@ export default function AddLiquidity() {
   const currencyA = useCurrency(currencyIdA)
   const currencyB = useCurrency(currencyIdB)
 
+  useEffect(() => {
+    if (zapMode !== temporarilyZapMode) {
+      setTemporarilyZapMode(zapMode)
+    }
+  }, [zapMode, temporarilyZapMode, setTemporarilyZapMode])
```

Comment:
when zapMode is true and temporarilyZapMode is false
it will set temporarilyZapMode back to true?